bindings/node/test/autoEncrypter.test.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+const fs = require('fs');
+const BSON = new (require('bson'))();
+const EJSON = require('mongodb-extjson');
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const StateMachine = require('../lib/stateMachine').StateMachine;
+
+const SegfaultHandler = require('segfault-handler');
+SegfaultHandler.registerHandler();
+
+function readExtendedJsonToBuffer(path) {
```

Comment:
nit/note: we should add `EJSON.toBuffer` and `EJSON.fromBuffer` methods to `bson`

Reply:
:+1:, ticket?

Reply:
https://jira.mongodb.org/browse/NODE-2008